doc/decisions/frontend_timeline_concept.md

```diff
@@ -0,0 +1,25 @@
+# Timeline: Custom Date Picker Implementation Decision
+
+## Problem Statement
+
+To implement the timeline feature, we require a custom date picker capable of displaying indicators for plant additions and removals.
+
+## Proposed Solutions
+
+Initially, the team considered using [ReactCharts](https://react-charts.tanstack.com/), a powerful charting library. However, it is not designed for date selection. As a result, we have decided to create custom components for building the timeslider:
```

Comment:
Please use template and more details what means "not designed for date selection"? What exact problems did you have.

EDIT: I later saw it is described below.